### PR TITLE
fixed schedule currency_rates_update cron job not working

### PR DESCRIPTION
**Pull request by brunoroeder**
### Description (*)

Fixed the schedule currency_rates_update cron job. 

Screenshot at Sep 14 12-14-48

### Fixed Issues (if relevant)

1. #23846: Schedule currency_rates_update cron job not working

### Manual testing scenarios (*)

    1. Go to admin panel
    2. Access: Stores > Configuration > General Tab > Currency Setup > Scheduled Import Setting section
    Set values:
       -- Enable -> Set Yes
       -- Service -> Fixer.io
       -- Start Time -> Fill current time + 5 min
       -- Frequency -> Choose Daily
    3. Clear the Magento cache
    4. Run php bin/magento cron:run from command line
    5. The currency_rates_update cron job exists in cron_schedule table

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by brunoroeder** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @brunoroeder. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @brunoroeder, here is your [new Magento instance](https://pr-24590.instances.magento-community.engineering).
Admin access: [https://pr-24590.instances.magento-community.engineering/admin](https://pr-24590.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/code/Magento/Directory/Model/Observer.php [hunk 1]
```diff
@@ -83,12 +88,13 @@ public function __construct(
...
     /**
-     * @param mixed $schedule
+     * Schedule update currency rates
+     *
      * @return void
      * @throws \Exception
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function scheduledUpdateCurrencyRates($schedule)
+    public function scheduledUpdateCurrencyRates()
```
Comment (sidolov) [inline app/code/Magento/Directory/Model/Observer.php]: Unfortunately, we are not able to change the public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

**Review by sidolov — CHANGES_REQUESTED** [review]

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5934](https://jira.corp.magento.com/browse/ENGCOM-5934) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**